ALLHUBS-Jan-2024-Liftoff/code-stars
# 14 review form

I got the basic review form up and running. Here's what I did:
- Set up the review form to submit to the backend using Axios
- Encapsulated the form in its own component
- Updated the CrossOrigin annotation in the review controller to fix CORS issues
- Changed the Review model's review field to "feedback" to prevent variable shadowing

To test my branch:
1. Run npm install in VSCode
2. Start all servers
3. Create a campground using the form
4. Submit a review using the corresponding campground id (1)
5. Check that the review was created in Workbench

Note that the review will not include a user ID. We can implement that once account creation and management is set up. In the future, I plan to replace the campground id text entry field with a dropdown, listing the campgrounds in our database.
